games-emulation/jrommanager/jrommanager-1.6-r1.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN=JRomManager
+MY_PV=1.6b20
```

Comment:
Do not ever hardcode the verson inside ebuild. Instead, adjust filename so that all version components are included, and then transform it as necessary for MY_PV. For example, if 'b' means 'build' here (and not beta), you could e.g. make it `1.6_p20` (which is a valid ebuild version), and then use a trivial bash substitution to replace `_p` with `b`.

Reply:
ok will do it